vm/src/vm/errors/vm_exception.rs

```diff
@@ -256,6 +256,14 @@ impl Location {
     pub fn to_string_with_content(&self, message: &str) -> String {
         let mut string = self.to_string(message);
         let input_file_path = std::path::Path::new(&self.input_file.filename);
+        #[cfg(test)]
+        let input_file_path = {
+            use std::path::PathBuf;
+            let current_dir = std::env::current_dir().unwrap();
+            let mut parent_dir: PathBuf = current_dir.parent().unwrap().into();
+            parent_dir.push(input_file_path);
+            parent_dir
+        };
```

Comment:
This fixes the path problem.
As tests are run from the crate directory itself, we can just get the current_dir, take its parent, and then append the `input_file_path`.

It gives us an absolute path that works.
That doesn't solve the case where programs are not compiled from the same dir as the one the vm is executed. But it will be a problem for later

Reply:
That's cool. Could you add that information as a little comment in place?